packages/agoric-cli/src/scripts.js

```diff
@@ -161,8 +161,15 @@ export { bootPlugin } from ${JSON.stringify(absPath)};
 
       const allEndowments = harden({
         home: bootP,
-        bundleSource: (file, options = undefined) =>
-          bundleSource(pathResolve(file), options),
+        /**
+         * @param {string} file
+         * @returns {Promise<import('@endo/bundle-source').BundleSourceResult<'endoZipBase64'>>}
+         */
+        bundleSource: file =>
+          bundleSource(pathResolve(file), {
+            elideComments: true,
+            format: 'endoZipBase64',
+          }),
```

Comment:
This should preserve the options argument. I think the trick is to just type this as `typeof import('@endo/bundle-source').bundleSource`, or instead of this, pass `elideComments` from the various call sites where the `format` is known.
```suggestion
        bundleSource: (file, options = {}) =>
          bundleSource(pathResolve(file), {
            elideComments: true,
            format: 'endoZipBase64',
            ...options
          }),
```